yellowbrick/model_selection/dropping_curve.py

```diff
@@ -0,0 +1,379 @@
+        Possible inputs for cv are:
+          - None, to use the default 3-fold cross-validation,
+          - integer, to specify the number of folds.
+          - An object to be used as a cross-validation generator.
+          - An iterable yielding train/test splits.
+        see the scikit-learn
+        `cross-validation guide <https://bit.ly/2MMQAI7>`_
+        for more information on the possible strategies that can be used here.
+    scoring : string, callable or None, optional, default: None
+        A string or scorer callable object / function with signature
+        ``scorer(estimator, X, y)``. See scikit-learn model evaluation
+        documentation for names of possible metrics.
+    n_jobs : integer, optional
+        Number of jobs to run in parallel (default 1).
+    pre_dispatch : integer or string, optional
+        Number of predispatched jobs for parallel execution (default is
+        all). The option can reduce the allocated memory. The string can
+        be an expression like '2*n_jobs'.
+    random_state : int, RandomState instance or None, optional (default=None)
+        If int, random_state is the seed used by the random number generator;
+        If RandomState instance, random_state is the random number generator;
+        If None, the random number generator is the RandomState instance used
+        by `np.random`. Used to generate feature subsets.
+    kwargs : dict
+        Keyword arguments that are passed to the base class and may influence
+        the visualization as defined in other Visualizers.
+    Attributes
+    ----------
+    feature_sizes_ : array,     shape = (n_unique_ticks,), dtype int
+        Numbers of features that have been used to generate the
+        dropping curve. Note that the number of ticks might be less
+        than n_ticks because duplicate entries will be removed.
+    train_scores_ : array, shape (n_ticks, n_cv_folds)
+        Scores on training sets.
+    train_scores_mean_ : array, shape (n_ticks,)
+        Mean training data scores for each training split
+    train_scores_std_ : array, shape (n_ticks,)
+        Standard deviation of training data scores for each training split
+    valid_scores_ : array, shape (n_ticks, n_cv_folds)
+        Scores on validation set.
+    valid_scores_mean_ : array, shape (n_ticks,)
+        Mean scores for each validation split
+    valid_scores_std_ : array, shape (n_ticks,)
+        Standard deviation of  scores for each validation split
+    Examples
+    --------
+    >>> from yellowbrick.model_selection import DroppingCurve
+    >>> from sklearn.naive_bayes import GaussianNB
+    >>> model = DroppingCurve(GaussianNB())
+    >>> model.fit(X, y)
+    >>> model.show()
+    Notes
+    -----
+    This visualizer is based on sklearn.model_selection.validation_curve
+    """
+
+    def __init__(
+        self,
+        estimator,
+        feature_sizes=DEFAULT_FEATURE_SIZES,
+        groups=None,
+        ax=None,
```

Comment:
Other [Yellowbrick model visualizers](https://github.com/DistrictDataLabs/yellowbrick/blob/develop/yellowbrick/classifier/rocauc.py#L66-L76) have the `estimator` then the `ax` as the first two arguments, saving visualizer-specific arguments until after these positions (but leaving `**kwargs` last). Although `ax` is not a positional argument, this inconsistency may cause users of this visualizer to make mistakes when shortcutting the call with positional args.